certbot/client.py

```diff
@@ -52,9 +53,12 @@ def determine_user_agent(config):
     """
 
     if config.user_agent is None:
-        ua = "CertbotACMEClient/{0} ({1}) Authenticator/{2} Installer/{3}"
+        auto = "no" if cli.cli_command == "certbot" else "yes"
```

Comment:
@hlieberman I believe that `cli.cli_command` would be set to `"letsencrypt-auto"` for those people, so for them this variable would be `"yes"`.